pages/market.tsx

```diff
@@ -60,7 +66,17 @@ const Market: MarketComponent = () => {
 						<LeftSidebar />
 						<MarketInfo />
 						<StyledRightSideContent>
+							<TradePanelHeader
+								onManageBalance={() => dispatch(setOpenModal('futures_isolated_transfer'))}
+								accountType={'isolated_margin'}
```

Comment:
Can we move this back, as it is specific to IsolatedMargin?

Reply:
Sure. I will revert this change. Let's minimize the change when introducing new features. My initial thought was to show the `deposit margin` when the user connected to our market page from a non-optimism chain. So they can start onboarding immediately by opening the `deposit margin` and bridging the token to optimism.

Reply:
Yea I see what you mean but I think this would need some more thought around correct UX. So best to leave how it was for now and we can create a design ticket if we think this approach makes sense.

Reply:
Reverted. Thanks.